DuckDuckGo/SuggestionTrayViewController.swift

```diff
@@ -107,7 +109,12 @@ class SuggestionTrayViewController: UIViewController {
         backgroundView.layer.shadowRadius = 120
 
         topConstraint.constant = 15
-        variableHeightConstraint.constant = 276
+        
+        let isFirstPresentation = fullHeightConstraint.isActive
+        if isFirstPresentation {
+            variableHeightConstraint.constant = 46 * 6
```

Comment:
nitpick: since 276 has been extracted in to a kind of derived value, should `46` be pulled out as a constant, like `iconSize` or something?